bench/parser.go

```diff
@@ -98,7 +98,8 @@ func (p *Parser) readBenchmark(line string) (*Benchmark, error) {
 
 	// runs - doesn't include units
 	tmp, split = internal.Popleft(split)
-	if bench.Runs, err = strconv.Atoi(tmp); err != nil {
+    // Ignore CPU information for now, until we support parsing it: https://github.com/bobheadxi/gobenchdata/issues/47
+	if bench.Runs, err = strconv.Atoi(tmp); err != nil && !strings.Contains(line, "cpu:") {
```

Comment:
Also I just realized this entire check should be added here https://sourcegraph.com/github.com/bobheadxi/gobenchdata/-/blob/bench/parser.go?L69 instead - notice how that's where we check for other special data prefixes